docs/content/docs/2.components/input-menu.md

```
@@ -252,6 +255,24 @@ props:
 ---
 ::
 
+### Autocomplete :badge{label="Soon" class="align-text-top"}
+
+Use the `autocomplete` prop to turn the InputMenu into a free-form text input with suggestions. The `modelValue` becomes the input text (`string`) instead of a selected item.
+
+::component-example
+---
+name: 'input-menu-autocomplete-example'
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)